CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.5 (2019-01-09)
+
+- Fixed card component to trigger infinite loops when selected. [#12](https://github.com/blackbaud/skyux-layout/pull/12)
```

Comment:
to "not" trigger?

Reply:
I'd change "trigger" to "prevent." ... Also, does the loop occur when the card component is selected? I thought it occurred when a checkbox within the card component was selected. If so, I'd add that to the end: "... infinite loops when users select a checkbox."

Reply:
Doh. Fixed